src/_pytest/config/findpaths.py

```diff
@@ -64,9 +64,9 @@ def load_config_dict_from_file(
 
     # '.toml' files are considered if they contain a [tool.pytest.ini_options] table.
     elif filepath.suffix == ".toml":
-        import toml
+        import tomli
 
-        config = toml.load(str(filepath))
+        config = tomli.loads(filepath.read_text(encoding="utf-8"))
```

Comment:
Not directly related to your PR, but more of a question to @nicoddemus: I wonder why we don't handle `tomli.TOMLDecodeError` here and turn it into an `UsageError`, like we do with `_parse_ini_config` above?

Reply:
Good idea, i think it should be done

Its up to @hukkin whether this is ok for this pr or if it will be a follow-up

Reply:
I made the change (as it's a very small change). Is this something we want a test for btw?

Reply:
>  Is this something we want a test for btw?

Well I thought it couldn't do any harm, so added a test for the exception type.

Reply:
> Not directly related to your PR, but more of a question to @nicoddemus: I wonder why we don't handle tomli.TOMLDecodeError here and turn it into an UsageError, like we do with _parse_ini_config above?

Probably it was an oversight only, good catch!